style.css

```diff
@@ -1,3 +1,429 @@
-h1 {
-  color: green;
+body {
+  background: #e5e5e5;
+  background-image: ./hello/images/header-shapes.png;
+}
+
+.hex {
+  color: #6070ff;
+}
+
+a:hover {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.main-wrapper {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  border-bottom-left-radius: 80px;
+}
+
+.toolbar {
+  flex: 1 1 auto;
+  display: flex;
+  height: 48px;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 40px;
+}
+
+.logo-place {
+  margin-left: 12px;
+}
+
+.my-logo {
+  padding: 10px 12px 10px 12px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.headline {
+  width: 100%;
+  background-image: url("/images/header-shapes.png");
+  background-size: 380px 550px;
+  border-radius: 0 0 0 100px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  padding: 20px 10px 0 10px;
+  height: calc(718px - 60px);
+  background-repeat: no-repeat;
+  overflow: hidden;
+}
+
+.headline > * {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.headline h1 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 35px;
+  letter-spacing: 0.25px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  padding-top: 80px;
+}
+
+.text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  text-align: justify;
+  text-justify: inter-word;
+  width: 327px;
+  height: 150px;
+  padding-right: 20px;
+}
+
+.connect {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.social-media {
+  display: flex;
+  width: 170.17px;
+  height: 20px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-bottom: 136px;
+  list-style-type: none;
+  padding-left: 15px;
+}
```

Comment:
- [ ] Your UI looks great and almost perfect. 👍 Just you need to fix the alignment of the icons with the `title` and `paragraph` sections. The icons seem a bit outside of the alignment. Kindly fix this to match the given figma design.

- Your design
- ![image](https://user-images.githubusercontent.com/60083437/227652363-d03a521c-9a45-431f-bffa-f97e2ab4d0de.png)

- Given Figma Design
- ![image](https://user-images.githubusercontent.com/60083437/227652482-70cada04-c9f5-4870-ab40-79bfdf950cf9.png)